AUTHORS

```diff
@@ -518,16 +560,18 @@ Peter Braden <peterbraden@peterbraden.co.uk>
 Peter Ericson <pdericson@gmail.com>
 Peter Salvatore <peter@psftw.com>
 Peter Waller <p@pwaller.net>
+Phil <underscorephil@gmail.com>
 Phil Estes <estesp@linux.vnet.ibm.com>
+Phil Spitler <pspitler@gmail.com>
 Philipp Weissensteiner <mail@philippweissensteiner.com>
 Phillip Alexander <git@phillipalexander.io>
-Phil Spitler <pspitler@gmail.com>
-Phil <underscorephil@gmail.com>
 Piergiuliano Bossi <pgbossi@gmail.com>
-Pierre-Alain RIVIERE <pariviere@ippon.fr>
 Pierre <py@poujade.org>
+Pierre Wacrenier <pierre.wacrenier@gmail.com>
+Pierre-Alain RIVIERE <pariviere@ippon.fr>
 Piotr Bogdan <ppbogdan@gmail.com>
 pixelistik <pixelistik@users.noreply.github.com>
+Porjo <porjo38@yahoo.com.au>
```

Comment:
`Ian Bishop <ianbishop@pace7.com>` I'm pretty certain these are the same since google searching shows other projects being sign by `Ian B <porjo38@yahoo.com.au>`
https://github.com/docker/docker/commits?author=Porjo